en/installation-guide/database-initialization.rst

```diff
@@ -40,19 +29,7 @@ Restore the latest data model that also includes demo data:
   * https://github.com/QGEP/datamodel/releases/latest
   * download `qgep_vx.y.z_structure_and_demo_data.backup`
   
-* Back in pgAdmin, create a new database with UTF8 encoding. 
-
-.. note:: 
-
- For restoring demo datamodel, the database has to be named `qgep_prod`.
-
-* Change to this database
-
-* Open an :ref:`SQL-query` Window and create the extensions (if they're not created yet)
-
-  * ``CREATE EXTENSION hstore;``
-
-  * ``CREATE EXTENSION postgis;``
```

Comment:
I think the postgis extension is still required.

Reply:
I went step by step on PgAdmin4 and did not encounter any troubles not creating these before restoring the dumps. (extensions were created anyway)

what do you think about short animated steps (gifs) to guide the user in quick installation steps?

Reply:
Then let's keep it as is.

For the videos, that would be nice!